### PR TITLE
337 websockets

### PR DESCRIPTION
**Pull request by jellegerbrandy**

### CODE REVIEW

File: src/components/Proposal/ProposalContainer.tsx [hunk 1]
```diff
@@ -472,14 +471,13 @@ class ProposalContainer extends React.Component<IProps, IState> {
...
                   proposal={proposal}
-                  stakeProposal={stakeProposal}
                   threshold={threshold}
                   approveStakingGens={approveStakingGens}
                   detailView={detailView}
                 />
               }
           </div>
-        </div>
+        </div >
```
Comment (brodeur) [inline src/components/Proposal/ProposalContainer.tsx]: Whitespace ^_^

File: src/components/Shared/PreTransactionModal.tsx [hunk 2]
```diff
@@ -211,7 +198,7 @@ export default class PreTransactionModal extends React.Component<IProps, IState>
             </div>
             <div className={css.proposalInfo}>
               <div className={css.proposalTitle}>
-                <strong>{proposal.title || "[no title]"}</strong>
+                <strong>{proposal.title || proposal.id}</strong>
```
Comment (brodeur) [inline src/components/Shared/PreTransactionModal.tsx]: Isn't proposal.id a hash? Is that better to show than "No title"? Could also be: "Title unavailable"

File: src/components/Proposal/PredictionBox.tsx [hunk 3]
```diff
@@ -224,7 +230,7 @@ export default class PredictionBox extends React.Component<IProps, IState> {
           </div>
 
           <div className={css.enablePredictions}>
-            <button onClick={this.showApprovalModal.bind(this)}>Enable Predicting</button>
+            <button onClick={this.showApprovalModal }>Enable Predicting</button>
```
Comment (brodeur) [inline src/components/Proposal/PredictionBox.tsx]: Whitespace

File: src/components/Shared/PreTransactionModal.tsx (hunk 2 above)
Comment (A-Zak) [reply src/components/Shared/PreTransactionModal.tsx]: Can we please combine the two but only using part of the id?
Like this - [No title 0xc839...8eee]

Comment (jellegerbrandy) [reply src/components/Shared/PreTransactionModal.tsx]: Ok

**Pull request merged**